app/schemas.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+
+from fastapi import Query
+from pydantic import BaseModel, Field
+
+
+class ARCutoutClassificationInput(BaseModel):
+    time: datetime = Field(Query(title='Date time', ge=datetime(2011, 1, 1), le=datetime.now()),
```

Comment:
Is there a specific reason for `2011, 1, 1`

Reply:
Very arbitrary - the plan is to ultimately makes plot as well and had initially thought about supporting both HMI and MDI but figured might be easier to focus on HMI and I figured there should be HMI data after that date.